hawkular-alerts-engine/pom.xml

```diff
@@ -88,6 +88,12 @@
       <version>${version.com.google.guava}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.reactivex</groupId>
+      <artifactId>rxjava</artifactId>
```

Comment:
> I don't have anything against RxJava
> I'm not so sure I understand the "mixed technology" thing, it's just a java API.

I don't have anything against RxJava, Spring, Netty, Vertx to cite some examples of Java APIs.
My comment was more related to the point that in Java EE some of those networks can overlaps resources and pattern than are available in Wildfly. 
Mixing Java EE and non EE technologies work but at some cost, I tend to maximize the use of the Java EE resources unless there is a blocker/need for additional things.


> Regardless of impl, one thing in the PR that I want to promote is the idea of IncomingDataManager, or basically a component where we can perform various filtering/throttling/massaging of the incoming data in an async way.

I got that. #244 is an approach of this idea with an alternative implementation.